simple-haskell-marker/IncrementSerializedClassifier.py

```diff
@@ -1,12 +1,59 @@
 import os
 import subprocess
 import sys
+import random
+import datetime
+import argparse
+
+# ARG PARSER ###########################################################
+parser = argparse.ArgumentParser()
+    
+parser.add_argument('--clean',
+                    action='store_true',
+                    default=False
+                    )
+
+args = parser.parse_args()
+
+# UTILITY FUNCTIONS ####################################################
+
+def datetime_past_seconds(time):
+    now = datetime.datetime.now()
+    difference = now - time
+    return difference.total_seconds()
+
+def clear_directory(target):
+    old_threshold = 3600 # Delete files older than 1 hour
+    randint = random.randint(0,100) # 1 possibility in 100 of performing a cleaning
```

Comment:
Couldnt we just have a scheduler? Once every 100 times, but deterministic rather than probabilistic and with less complexity overhead?